src/workers/cdc.ts

```diff
@@ -161,6 +164,35 @@ const onAlertsChange = async (
   }
 };
 
+const hasAnyChanged = <T>(before: T, after: T, excludes: (keyof T)[] = []) =>
+  Object.keys(before).some((key) => {
+    if (excludes.some((excluding) => excluding === key)) {
+      return false;
+    }
+
+    return before[key] !== after[key];
+  });
+
+const onSettingsChange = async (
+  _: Connection,
+  logger: FastifyLoggerInstance,
+  data: ChangeMessage<Settings>,
+): Promise<void> => {
+  if (data.payload.op === 'u') {
+    const hasChanged = hasAnyChanged(
+      data.payload.before,
+      data.payload.after,
+      excludedSettingsProps,
+    );
```

Comment:
I think this is a bit redundant because practically `userId` will never be updated and `updatedAt` will be updated only when another field is being updated. So you can just fire the event on any update operation

Reply:
Oh, yes. I just checked. My mistake.